kafka-ui-react-app/src/components/Topics/Details/Messages/MessageItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { format } from 'date-fns';
+import JSONTree from 'react-json-tree';
+import { TopicMessage } from 'generated-sources';
+
+interface MessageItemProp {
+  partition: TopicMessage['partition'];
+  offset: TopicMessage['offset'];
+  timestamp: TopicMessage['timestamp'];
+  content: Record<string, unknown>;
+}
+
+const MessageItem: React.FC<MessageItemProp> = ({
+  partition,
+  offset,
+  timestamp,
+  content,
+}) => {
+  const getFormattedDate = (date: Date) => {
+    if (!date) return null;
+    return format(date, 'yyyy-MM-dd HH:mm:ss');
+  };
+
+  const getMessageContentBody = (messageContent: Record<string, unknown>) => {
+    try {
+      const contentObj =
+        typeof messageContent !== 'object'
+          ? JSON.parse(messageContent)
+          : messageContent;
+      return (
+        <JSONTree
+          data={contentObj}
+          hideRoot
+          invertTheme={false}
+          theme={{
+            tree: ({ style }) => ({
+              style: {
+                ...style,
+                backgroundColor: undefined,
+                marginLeft: 0,
+                marginTop: 0,
+              },
+            }),
+            value: ({ style }) => ({
+              style: { ...style, marginLeft: 0 },
+            }),
+            base0D: '#3273dc',
+            base0B: '#363636',
+          }}
+        />
+      );
+    } catch (e) {
+      return messageContent;
+    }
+  };
+
+  return (
+    <tr key={`${timestamp}${Math.random()}`}>
```

Comment:
What about use [uuid](https://github.com/uuidjs/uuid#readme) lib instead of `Math.random`?

Reply:
I agree that we should get rid of Math.random. But disagree of using uuid here

Reply:
From my point of view, just a `timestamp` is enough